contracts/axelar-soroban-interfaces/src/types.rs

```diff
@@ -57,10 +63,15 @@ pub struct Message {
 }
 
 impl WeightedSigners {
-    // Get hash of WeightedSigners
     pub fn hash(&self, env: &Env) -> BytesN<32> {
         env.crypto().keccak256(&self.clone().to_xdr(env)).into()
     }
+
+    pub fn signers_rotation_hash(&self, env: &Env) -> BytesN<32> {
```

Comment:
good to add this method but we're trying to move away from adding things to the interface crate. i'll approve to unblock the golden test, but we should move away from this crate asap

Reply:
I see. I added `signers_rotation_hash` function similar to [the imp WeightedSigners](https://github.com/axelarnetwork/axelar-amplifier/blob/7decfc9b0f2277a362c621e2bb886b5005514094/external-gateways/stellar/src/lib.rs#L174) in axelar-amplifier. 
Then I'm wondering if we should move the `hash` function as well. If we only move `signers_rotation_hash`, the test code would look like the example below, which seems inconsistent.

```
        let hash = weighted_signers.hash(&env);
        let signers_rotation_hash = signers_rotation_hash(&env, weighted_signers.signers.clone());
```

Reply:
we'll be removing the interface crate altogether and move all the types to the appropriate crates. the usage is here fine until then

Reply:
Got it. thanks for clarifying it